src/main/java/com/ays/assignment/model/entity/AssignmentEntity.java

```diff
@@ -0,0 +1,94 @@
+package com.ays.assignment.model.entity;
+
+import com.ays.assignment.model.enums.AssignmentStatus;
+import com.ays.common.model.entity.BaseEntity;
+import com.ays.institution.model.entity.InstitutionEntity;
+import com.ays.user.model.entity.UserEntity;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * Assignment entity, which holds the information regarding assignment.
+ */
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "AYS_USER_ASSIGNMENT")
+public class AssignmentEntity extends BaseEntity {
+
+    @Id
+    @Column(name = "ID")
+    private String id;
+
+    @Column(name = "INSTITUTION_ID")
+    private String institutionId;
+
+    @Column(name = "USER_ID")
+    private String userId;
+
+    @Column(name = "DESCRIPTION")
+    private String description;
+
+    @Column(name = "FIRST_NAME")
+    private String firstName;
+
+    @Column(name = "LAST_NAME")
+    private String lastName;
+
+    @Column(name = "COUNTRY_CODE")
+    private String countryCode;
+
+    @Column(name = "LINE_NUMBER")
+    private String lineNumber;
+
+    @Column(name = "LATITUDE")
+    private Double latitude;
+
+    @Column(name = "LONGITUDE")
+    private Double longitude;
```

Comment:
Veritabanı katmanında `POINT` veri tipine sahip bir alan kullanacağımız için burada da `latitude` ve `longtitude` olarak ayrı ayrı tutmak yerine `org.locationtech.jts.geom.Point` objesini kullanırsak kendi içerisinde `latitude` ve `longtitude` bilgilerini de tutabiliyor olacak ve hesaplamaları kolay bir şekilde gerçekleştirebileceğiz

Reply:
Gerekli değişiklikleri yaptım. Kontrol edebilir misin? @agitrubard